src/__tests__/NonNegativeFloat.test.js

```diff
@@ -232,3 +232,232 @@ describe('NonNegativeFloat', () => {
     });
   });
 });
+
+describe('UnsignedFloat', () => {
+  describe('valid', () => {
+    describe('greater than zero', () => {
+      describe('as float', () => {
+        test('serialize', () => {
+          expect(UnsignedFloat.serialize(123.45)).toBe(123.45);
+        });
+
+        test('parseValue', () => {
+          expect(UnsignedFloat.parseValue(123.45)).toBe(123.45);
+        });
+
+        test('parseLiteral', () => {
+          expect(
+            UnsignedFloat.parseLiteral({ value: 123.45, kind: Kind.FLOAT }),
+          ).toBe(123.45);
+        });
+      });
+
+      describe('as string', () => {
+        test('serialize', () => {
+          expect(UnsignedFloat.serialize('123.45')).toBe(123.45);
+        });
+
+        test('parseValue', () => {
+          expect(UnsignedFloat.parseValue('123.45')).toBe(123.45);
+        });
+
+        test('parseLiteral', () => {
+          expect(
+            UnsignedFloat.parseLiteral({
+              value: '123.45',
+              kind: Kind.FLOAT,
+            }),
+          ).toBe(123.45);
+        });
+      });
+    });
+
+    describe('zero', () => {
+      describe('as float', () => {
+        test('serialize', () => {
+          expect(UnsignedFloat.serialize(0.0)).toBe(0.0);
+        });
+
+        test('parseValue', () => {
+          expect(UnsignedFloat.parseValue(0.0)).toBe(0.0);
+        });
+
+        test('parseLiteral', () => {
+          expect(
+            UnsignedFloat.parseLiteral({ value: 0.0, kind: Kind.FLOAT }),
+          ).toBe(0.0);
+        });
+      });
+
+      describe('as string', () => {
+        test('serialize', () => {
+          expect(UnsignedFloat.serialize('0.0')).toBe(0.0);
+        });
+
+        test('parseValue', () => {
+          expect(UnsignedFloat.parseValue('0.0')).toBe(0.0);
+        });
+
+        test('parseLiteral', () => {
+          expect(
+            UnsignedFloat.parseLiteral({ value: '0.0', kind: Kind.FLOAT }),
+          ).toBe(0.0);
+        });
+      });
+    });
+  });
+
+  describe('invalid', () => {
+    describe('null', () => {
+      test('serialize', () => {
+        expect(() => UnsignedFloat.serialize(null)).toThrow(
+          /Value is not a number/,
+        );
+      });
+
+      // FIXME: Does nothing. No throw. Call doesn't even seem to get to the parseValue() function.
```

Comment:
Is this still occurring?

Reply:
Cleaned up several. Some comments still remain because these are still issues.